pkg/exchange/bitget/exchange.go

```diff
@@ -113,14 +113,17 @@ func (e *Exchange) QueryTicker(ctx context.Context, symbol string) (*types.Ticke
 		return nil, fmt.Errorf("ticker rate limiter wait error: %w", err)
 	}
 
-	req := e.client.NewGetTickerRequest()
+	req := e.v2Client.NewGetTickersRequest()
```

Comment:
how about v2client? all in lower case

Reply:
ok, update in next

Reply:
Ah, I updated this PR because the checks were not working after another PR was merged...